test/verify/pytest_dynamic_loader.py

```diff
@@ -1,7 +1,10 @@
 import importlib
 import glob
 import os
-from inspect import isclass
```

Comment:
I don't have a huge attachment to pytest_dynamic_loader.py myself, as we don't use it at all upstream nor in Fedora gating. However, it seems a bit weird to me to combine "FMF formatter" and "pytest runner" in one and the same file? Aren't these two wildly different purposes? Also, do you still actually need the pytest loader once you move to FMF?

Reply:
already reworked to separated file.

If I need this pytest loader, yes and no :-)

from one perspective, run-test does lots of magic inside, e.g. rescheduling, skipping logic, which is perfect but theoretically it maybe sometimes contra-productive and can hide potential issues (e.g. skip some test what is expected to be scheduled because it is important) also this rescheduling may lead to some strange situation. or may hide issue that it sometimes fails because of some error.